charts/ai-software-templates/pipeline-install/templates/app-config.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name }}-app-config
+    app.kubernetes.io/name:  {{ .Release.Name }}-app-config
+    app.kubernetes.io/part-of: {{ .Release.Name }}
+  name: {{ .Release.Name }}-app-config
+  namespace: {{ .Release.Namespace }}  
+data:
+  APP_NAME: "{{ .Release.Name }}"
+  APP_NAMESPACE: "{{ .Release.Namespace }}"
+  GITHUB_APP_APP_ID: "{{ .Values.tekton.gitAppId }}"
+  GITHUB_APP_WEBHOOK_SECRET: "{{ .Values.tekton.gitAppWebhookSecret }}"
+  GITHUB_APP_WEBHOOK_URL: "{{ .Values.tekton.gitAppWebhookURL }}"
+  GITHUB_APP_PRIVATE_KEY: "{{ .Values.tekton.gitAppPrivateKey }}"
```

Comment:
per conversation with @coreydaley I should  move the private key to a Secret and set the env var in the Job from that

I'll work on that tomorrow, but wanted to get the PR up since there is plenty else to review

Reply:
actually this is moot because of the helm pre-hook deletion policy I employ .... this config map will be deleted per the setting here:  https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/28/files#diff-4fe737cf1f57cce8ef7854a2ca630dea36a68fae81d14935c57c95f7174ddcd7

Reply:
Does it make sense to create an extra secret resource just to pass those values encoded to the cluster (appId, webhookSecret and privateKey)?